rbd/errors.go

```diff
@@ -73,6 +73,8 @@ var (
 
 // Public general error
 const (
+	// ErrExist indicates a non-specific already existing resource.
+	ErrExist = rbdError(-C.EXISTS)
```

Comment:
```suggestion
	ErrExist = rbdError(-C.EEXIST)
```

Reply:
wow, and I thought I tested this :facepalm: